Assets/Mariana /Script/PlayerCollisionWEnemy.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class PlayerCollisionWEnemy : MonoBehaviour
+{
+
+    private void OnCollisionEnter2D(Collision2D collision)
+    {
+
+        if(collision.transform.tag == "Enemy")
+        {
+            SceneManager.LoadScene(2);
```

Comment:
This seems like a very specific thing to have it do.

Scene "2" might also change as this project develops, which might cause a conflict with other projects.